docs/source/blog/datashuttle.md

```diff
@@ -0,0 +1,220 @@
+is low on the list of priorities when it
+comes to acquisition sessions spent managing complex systems 
+and experimental animals.
+
+However, the cost of small mistakes during data acquisition can be high.
+One misplaced character may mean sessions are missed by analysis 
+scripts or subject identifications duplicated.
+The best protection against such errors is automating the process
+through acquisition scripts—resulting in hours spent writing code 
+managing the naming and transfer of folders—a task entirely 
+unrelated to the central research goals.
+
+In our previous blog, we highlighted the benefits of data standardisation 
+for systems neuroscience, introducing the 
+[NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/) 
+specification. 
+An immediate benefit of a widely-used standard is that the entire community
+can contribute to shared tools for project management.
+This means individual researchers don't waste time
+writing data-management code. 
+
+In this blog post we introduce 
+[**datashuttle**](https://datashuttle.neuroinformatics.dev/)—a 
+tool for the automated creation, 
+validation and transfer of projects organised to 
+the **NeuroBlueprint** standard. **datashutte** aims to
+drop into existing acquisition pipelines, reducing errors
+associated with manual folder creation and removing the need
+to write your down data-management code.
+
+Below we give a whistlestop tour of **datashuttles** and it's key
+features. More information is available at the 
+[datashutte](https://datashuttle.neuroinformatics.dev/) 
+website.
+
+## How **datashuttle** is used in an experiment
+
+```{image} /_static/blog_images/datashuttle/tutorial-1-example-file-tree-dark.png
+:align: center
+:class: only-dark
+:width: 400px
+```
+```{image} /_static/blog_images/datashuttle/tutorial-1-example-file-tree-light.png
+:align: center
+:class: only-light
+:width: 400px
+```
+<br>
+
+Imagine that you are starting a new experiment and have the first
+acquisition session, or behaviour (`behav`) 
+and electrophysiological (`ephys`) data. 
+
+The first thing typically done prior to acquiring data is to
+create the folders that the data will be stored. 
+**datashuttle** can be used to quickly create the standardised
+project folders in which to store the acquired data. 
+Using **datashuttle** as apposed to manual creation or using custom
+scripts is that it ensures no formatting errors creep in.
+
+At the end of the session once data is acquired, 
+**datashuttle** transfers the newly acquired data 
+to a central storage machine to be backed up.
+
+Later on in the experiment, you may want to transfer only a subset
+of data from the central machine to an analysis machine. You may 
+want to pilot a behavioural data analysis, and grab
+for example only the behavioural 'test' session for
+the first 5 subjects. **datashuttle** allows flexible custom transfers
+easily, meaning you don't have to drag and drop these data manually or
+write a custom script.
```

Comment:
Would it be worth rephrasing this as:

When organising data manually you would
* A (which can cause error X)
* B (which can cause error Y)
* C (which can cause error Z)

With datashuttle you can:
*
*

Reply:
I like the idea of laying it out like that. One issue that is a general awkwardness of this blogpost is it is trying to cater to both those who do data management manually as well as automated. The only worry about emphasising the (many) potential errors caused by manual too much is people who are using scripts think 'well, that's not a problem for me' and ignore it.

Reply:
Oh yeah, I hadn't thought about our two "target markets!